src/components/messaging.js

```diff
@@ -6,6 +6,7 @@ import biscuitMug from '../images/chicken-biscuit-mug.jpg'
 const Messaging = () => (
   <div className="container">
     <div className="d-md-inline-flex row align-items-center">
+    <h1>Open for <a href={config.userLinks[4].url}>Takeout</a> and <a href={config.userLinks[6].url}>Delivery</a>!</h1>
```

Comment:
This is the cause of the build problem.

Anytime `{config. }` is used `config.js` must be imported `import config from '../../config'`.